README.md

```diff
@@ -108,9 +108,10 @@ The primary export of `deep-eql` is function that can be given two objects to co
 - All own and inherited enumerable properties are considered:
   - `eql(Object.create({ foo: { a: 1 } }), Object.create({ foo: { a: 1 } })).should.be.true;`
   - `eql(Object.create({ foo: { a: 1 } }), Object.create({ foo: { a: 2 } })).should.be.false;`
+- When comparing `Error` objects, `name` and `message` properties are also considered, regardless of enumerability:
+  - `eql(Error('foo'), Error('foo')).should.be.true;`
+  - `eql(Error('foo'), Error('bar')).should.be.false;`
```

Comment:
Maybe worth using an example of an error with a property being set explicitly different, such as `code`, but with the same `message` and `type` being `false`.